distmaker/distmaker.sh

```diff
@@ -158,7 +201,7 @@ esac
 
 ## Make sure we have the right branch or tag
 dm_git_checkout "$DM_SOURCEDIR" "$DM_REF_CORE"
-dm_git_checkout "$DM_SOURCEDIR/packages" "$DM_REF_PACKAGES"
+dm_git_checkout "$DM_PACKAGESDIR" "$DM_REF_PACKAGES"
```

Comment:
That was the idea - it turned out to be most of the way to possible but it still doesn't quite complete with a diff structure